samples/bannerSample/src/main/java/com/inmobi/banner/sample/BannerAdsActivity.java

```diff
@@ -68,56 +72,60 @@ protected void onCreate(Bundle savedInstanceState) {
     }
 
     private void setupBannerAd() {
-        mBannerAd = new InMobiBanner(BannerAdsActivity.this, PlacementId.YOUR_PLACEMENT_ID);
-        RelativeLayout adContainer = (RelativeLayout) findViewById(R.id.ad_container);
-        mBannerAd.setAnimationType(InMobiBanner.AnimationType.ROTATE_HORIZONTAL_AXIS);
-        mBannerAd.setListener(new BannerAdEventListener() {
-            @Override
-            public void onAdLoadSucceeded(InMobiBanner inMobiBanner) {
-                super.onAdLoadSucceeded(inMobiBanner);
-                Log.d(TAG, "onAdLoadSucceeded");
-            }
+        try {
```

Comment:
`try/catch` block is not necessary for `SdkNotInitializedException` as the intention is to throw exception only for wrong integration. Publishers following the guidelines need not even handle this exception as it will never occur.